payments/mobile-pay/capture.md

```diff
@@ -0,0 +1,19 @@
+title: MobilePay Payments – Capture
```

Comment:
Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)